doc/sphinx/language/extensions/arguments-command.rst

```diff
@@ -50,51 +50,57 @@ Setting properties of a function's arguments
    it's not in a section.  Applying :attr:`global` within a section extends the
    effect outside the current sections and current module in which the command appears.
 
-      `/`
-         the function will be unfolded only if it's applied to at least the
-         arguments appearing before the `/`.  See :ref:`Args_effect_on_unfolding`.
```

Comment:
Would you put the entries back in their original order?  The entries are ordered to correspond to the order of the grammar productions.  Generally reordering should be done only if there's a good reason to do so--it makes it much harder to see what's been changed.  Looks like the original followed this principle imperfectly :-(.

Optional: `argument_spec` should be after `arg_specs` in the grammar

Reply:
Sorry, I forgot to explain that change. My goal was to make the order more consistent with the itemize just below the grammar. This indeed make things inconsistent with the grammar. I chose to fix that by reordering the grammar so that everything (grammar, itemize and precise description) is more consistent. Does that seem reasonable to you?

> [reordering] makes it much harder to see what's been changed

I agree, I tried to mitigate this by isolating the reordering in a dedicated commit.

> Optional: argument_spec should be after arg_specs in the grammar

Done